packages/hdwallet-native/src/kava.ts

```diff
@@ -0,0 +1,123 @@
+import * as core from "@shapeshiftoss/hdwallet-core";
+import { BIP32Interface } from "bitcoinjs-lib";
+import * as bitcoin from "bitcoinjs-lib";
```

Comment:
single import per package
```suggestion
import { BIP32Interface } from "bitcoinjs-lib";
import * as bitcoin from "bitcoinjs-lib";
```